### PR TITLE
Add 4.1.0 release notes to README.md

### PR DESCRIPTION
**Pull request by outdooracorn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,8 +90,11 @@ as [Wikimedia Germany](https://wikimedia.de) employee for the [Wikidata project]
 
 ### 5.0.0 (dev)
 
-* Updated minimal required PHP version from 5.5.9 to 7.0.
-  In particular, HHVM is no longer supported.
+### 4.1.0 (TBD)
+
+* Type hinted the `$previous` parameter as `Throwable` instead of `Exception`
+* Updated minimum required PHP version from `5.5.9` to `7.2` (HHVM is no longer supported)
```
Comment (mariushoch) [inline README.md:96]: This says that we support PHP from `5.5.9` to `7.2`... while it actually looks like we now support `7.2` to `8.0` (at least that's whats tested on travis)?

Comment (outdooracorn) [reply README.md:96]: It says we dropped `5.5.9` as the minimum supported version and that `7.2` is now the minimum supported version. Although, maybe this sentence isn't phrased very well. Feel free to suggest an alternative. :)

Comment (mariushoch) [reply README.md:96]: Huh, seems I simply overlooked the "minimum"... looking at it again, this seems all good.

**Review by mariushoch — APPROVED** [review]

**Review by micgro42 — APPROVED** [review]
Looks good to me as well. When this is merged, I could add a line for #34

**Comment by micgro42** [comment]
I'll be bold and merge this. so that we can move forward with the other PR

**Comment by outdooracorn** [comment]
Sorry, been busy moving house! Thanks for being bold :grinning:

**Pull request merged**